packages/tradershub/src/flows/FirstSignup/FirstSignupWrapper/FirstSignupWrapper.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useState } from 'react';
+import { Form, Formik } from 'formik';
+import ReactModal from 'react-modal';
+import { CUSTOM_STYLES } from '@/helpers';
+import { SignupScreens } from '../SignupScreens';
+
+export type TFirstSignupFormValues = {
+    citizenship: string;
+    country: string;
+    password: string;
+};
+
+const FirstSignupWrapper = () => {
+    const [isOpen] = useState(false);
+    const [step, setStep] = useState(1);
+
+    const initialValues = {
+        country: '',
+        citizenship: '',
+        password: '',
+    };
+
+    const handleSubmit = () => {
+        // will be added later
+    };
+
+    useEffect(() => {
+        ReactModal.setAppElement('#v2_modal_root');
+    }, []);
```

Comment:
If we're setting `ariaHideApp={false}` then we don't need to set the appElement. Can you please check by removing either or them?

Reply:
@shahzaib-deriv Thanks for this comment, I'll delete `ariaHideApp` prop